src/OpenTelemetry.Instrumentation.Wcf/Implementation/AspNetParentSpanCorrector.cs

```diff
@@ -59,7 +59,7 @@ private static void OnRequestStarted(Activity activity, object context)
         var request = RequestFetcher.Fetch(context);
         var headers = HeadersFetcher.Fetch(request);
 
-        ReflectedValues.SetHeadersReadOnly(headers, false);
+        ReflectedValues!.SetHeadersReadOnly(headers, false);
```

Comment:
I would probably add a `Debug.Assert` here too. Basically anywhere we are relying on something to be non-null but not actually validating. The goal being to catch any potential future mistakes during development.